src/App.tsx

```diff
@@ -48,11 +95,11 @@ export const App: React.FC = () => {
               'is-parent',
               'is-8-desktop',
               'Sidebar',
-              'Sidebar--open',
+              { 'Sidebar--open': selectedPost },
             )}
           >
             <div className="tile is-child box is-success ">
-              <PostDetails />
+              {selectedPost !== null && <PostDetails post={selectedPost} />}
```

Comment:
Just
```suggestion
              {!selectedPost && <PostDetails post={selectedPost} />}
```
Bc !null === true